docs/src/index.md

```diff
@@ -70,17 +68,18 @@ Coefficients:
 (Intercept)  -0.666667   0.62361 -1.06904   0.4788
 X                  2.5  0.288675  8.66025   0.0732
 
-julia> stderror(ols)
+julia> round.(stderror(ols), digits=5)
 2-element Array{Float64,1}:
  0.62361
- 0.288675
+ 0.28868
 
-julia> predict(ols)
+julia> round.(predict(ols), digits=5)
 3-element Array{Float64,1}:
  1.83333
  4.33333
  6.83333
-
+```
+ <!-- Andreas Noack: As of 9 May 2018 (and again 16 January 2019) this example (still) doesn't work so I've temporarily commented it out
```

Comment:
Apparently this syntax does not cause the lines to be commented out on the rendered html help.

Reply:
No. It seems that there isn't a way to comment out in our Markdown. It also messes up the rest of the file. I'm working on cleaning things up.